esp-wifi/MIGRATING-0.13.md

```diff
@@ -46,3 +46,14 @@ Some code can be simplified now.
 ## `AccessPointInfo` doesn't include the `protocols` field anymore
 
 That field was never populated and has been removed.
+
+## Deinitialization
```

Comment:
This should be moved to a new MG version

Reply:
We should rebase this on top of https://github.com/esp-rs/esp-hal/pull/3606